### PR TITLE
[Merged by Bors] - feat: add `OuterMeasure.ofFunction_eq_iInf_mem`

### PR DESCRIPTION
**Pull request by RemyDegenne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/OuterMeasure/OfFunction.lean [hunk 1]
```diff
@@ -90,6 +90,25 @@ theorem ofFunction_apply (s : Set α) :
     OuterMeasure.ofFunction m m_empty s = ⨅ (t : ℕ → Set α) (_ : s ⊆ iUnion t), ∑' n, m (t n) :=
   rfl
 
+/-- Same as `ofFunction_apply`, except that the sets `t i` satisfy a predicate `P`,
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/OuterMeasure/OfFunction.lean:93]: Can you also explain in the docstring what the theorem does, without referring to the other lemma? When they appear in tooltips, it's better if the docstrings are self-contained.

**Comment by sgouezel** [comment]
Can you fix the merge conflict?

**Comment by RemyDegenne** [comment]
Thanks for telling me. It would be nice to get an email notification in that case. The bot could post a comment on the PR for example.

**Comment by urkud** [comment]
Thanks! :tada:
bors merge

**Pull request closed without merging**